exceptions/strike.py

```diff
@@ -0,0 +1,43 @@
+"""Custom exceptions for strike related cogs."""
+
+from collections.abc import Sequence
+
+__all__: Sequence[str] = (
+    "StrikeTrackingError",
+    "NoAuditLogsStrikeTrackingError",
+)
+
+from classproperties import classproperty
+
+from exceptions import BaseTeXBotError
```

Comment:
Use relative imports for importing base exception classes. I.e. `from .base import ...` rather than ~`from exceptions import ...`~ or ~`from exceptions.base import ...`~